libraries/incubator/ORM/Entity/EntityBuilder.php

```diff
@@ -399,7 +397,7 @@ public function handleStorage(Element $storage)
 					break;
 			}
 
-			$this->reflector->setStorageProvider(new $handler($param));
```

Comment:
No it does not. All tests are passing, and when I change the the storage back to csv, then it is still working. I'v adapted the CSV provider as well https://github.com/joomla-projects/joomla-pythagoras/commit/6f2097f276ccc29633d101b68311d8d72aa0a604#diff-3e9334b69577f0bffbc0fe8d03ab094cR61